README.md

```diff
@@ -7,12 +7,12 @@
 This driver allows Kubernetes to use [Azure disk](https://azure.microsoft.com/en-us/services/storage/disks/) volume, csi plugin name: `disk.csi.azure.com`
 
 ### Container Images & Kubernetes Compatibility
-|Azure Disk CSI Driver Version  | Image                                              | 1.14+ |
-|-------------------------------|----------------------------------------------------|-------|
-|master branch                  |mcr.microsoft.com/k8s/csi/azuredisk-csi:latest      | yes   |
-|v0.8.0                         |mcr.microsoft.com/k8s/csi/azuredisk-csi:v0.8.0      | yes   |
-|v0.7.0                         |mcr.microsoft.com/k8s/csi/azuredisk-csi:v0.7.0      | yes   |
-|v0.6.0                         |mcr.microsoft.com/k8s/csi/azuredisk-csi:v0.6.0      | yes   |
+|Driver Version  |Image                                           | 1.14+ |
+|----------------|------------------------------------------------|-------|
+|master branch   |mcr.microsoft.com/k8s/csi/azuredisk-csi:latest  | yes   |
+|v0.8.0          |mcr.microsoft.com/k8s/csi/azuredisk-csi:v0.8.0  | yes   |
```

Comment:
@andyzhangx , shouldn't this to be `mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v0.8.0` instead of `mcr.microsoft.com/k8s/csi/azuredisk-csi:v0.8.0`? I see similar change for azurefile-csi-driver where the image repository is updated but here it is not - ref https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/395?

Reply:
@ialidzhikov we are in the process of moving release publish to `mcr.microsoft.com/oss/kubernetes-csi/` repo, currently these 2 repo addresses are both valid, after around 3 releases, will update the README page. Note that master branch always use `mcr.microsoft.com/k8s/csi/azuredisk-csi:latest` and `mcr.microsoft.com/oss/kubernetes-csi/` repo is only for published versions. Thanks.